Shared/css/style.css

```diff
@@ -9653,16 +9653,22 @@ margin:2px;
 }
 
 .toastText {
-  font-family: Arial; 
-  grid-area: "text";
-  overflow-y: scroll;
+  font-family: Arial;
   width: 100%;
-  min-height: 2rem;
-  height: 3rem;
+  min-height:2.3rem;
+  height: 2.3rem;
   word-break: break-word;
-  text-align: start; 
+  text-align: start;
+  overflow: hidden;
 }
 
+
```

Comment:
In the future, avoid having two whitespaces after another. Just one will do. For now you don't need to change this, just letting you know. Makes the code easier to read through for future students.